recipes/deepxde/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "DeepXDE" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b055b409255fca5efcde4e38bd5d4169f5a8fa0a7caac9fb03261cd54e650a4b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - matplotlib
+    - salib
+    - scikit-learn
+    - scipy
+    - tensorflow
+
+test:
+  imports:
+    # Skip the import test on Linux as TensorFlow require a more recent
+    # version of GLIBC++ than the VM used to build and test package.
+    - deepxde  # [not linux]
```

Comment:
If you are using selectors, this recipe cannot be noarch

Reply:
Thank you. Removed noarch.